src/stores/user.js

```diff
@@ -7,7 +7,8 @@ export const useUserStore = defineStore({
   state: () => ({
     user: {
       auth:null,
-      info:null
+      info:null,
+      docs:null
```

Comment:
do not call it 'docs' that implies that there are multiple when there are not.

Reply:
renamed it doc
![image](https://user-images.githubusercontent.com/46050460/233812102-08fa0c71-6ccb-498b-a246-bc40bb16a068.png)